nodes/ApifyContentCrawler/helpers/index.ts

```diff
@@ -1,2 +0,0 @@
-export * as hooks from './hooks';
```

Comment:
It looks that hooks.ts and consts.ts are not used anywhere. You can remove it most probably.